src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt

```diff
@@ -103,6 +103,10 @@ class XmppApi(
                     domain = config.controlLogin.domain
                     username = config.controlLogin.username
                     password = config.controlLogin.password
+                    port = config.controlLogin.port
+
+                    if (port.isNullOrEmpty())
+                        port = "5222"
```

Comment:
@bgrozev any thoughts on handling situations like this?  Wondering if we should define a `default-port` property in `reference.conf` or something to keep defaults more consistent, but I can't remember what we've done in other places.

Reply:
I don't think we'd ever need to change the default, so adding it to reference.conf doesn't seem appropriate. I would push it to JibiConfig.kt. We already default to 5222 down the line, so I would suggest to only set MucClientConfiguration.port if we have a non-default value